topostats/io.py

```diff
@@ -568,19 +612,26 @@ def load_topostats(self) -> tuple:
         LOGGER.info(f"Loading image from : {self.img_path}")
         try:
             with h5py.File(self.img_path, "r") as f:
-                keys = f.keys()
-                file_version = f["topostats_file_version"][()]
+                # Load the hdf5 data to dictionary
+                loaded_dictionary = hdf5_to_dict(open_hdf5_file=f, group_path="/")
```

Comment:
Perhaps a more appropriate name...

```suggestion
                topodata = hdf5_to_dict(open_hdf5_file=f, group_path="/")
```

**NB** IDE (VSCode) should be able to replace variable names across a whole project as its using Language Server Protocol in the background (I use this feature in Emacs and its great, and its like :magic_wand: as it changes it across all files things appear in).